frontend/src/components/ProgressBar/progressBar.less

```diff
@@ -0,0 +1,17 @@
+.progressBarWrapper {
+  width: '100%';
```

Comment:
css values do not have '' in the css value. i.e. should be `width: 100%` rather than `width: '100%'`

Reply:
Yeah forget to change it when moving it over from the react code. Fixed now